src/ZEO/ClientStorage.py

```diff
@@ -53,6 +53,9 @@
 
 logger = logging.getLogger(__name__)
 
+# max signed 64-bit value ~ infinity :) Signed cuz LBTree and TimeStamp
+m64 = b'\x7f\xff\xff\xff\xff\xff\xff\xff'
```

Comment:
Hmm, I don't see it:  maybe my memory played tricks on me, and the constant was used in part of your changes to ZODB today?